etl/docs/csv/import-schema.json

```
@@ -63,6 +63,12 @@
         "name": "benefits_end_month",
         "type": "yearmonth",
         "description": "The final month when a participant will receive benefits. Leave this field blank if the participant will continue receiving benefits until the next recertification period."
+      },
+      {
+        "name": "recent_benefit_months",
+        "type": "string",
+        "format": "%Y-%m %Y-%m %Y-%m",
```

If a CSV only has one or two dates, what will validate.py report against it?

---

It will not. I added a few more examples of <3 dates to the example csv.

---

Hmm... something isn't quite right, but not sure if it is in the schema, the tableschema code, or our validate.py. 

For example, try adding a bogus string like `xxyy` or `2021` to a `recent_benefit_months` cell in the CSV. Expectation is that it should throw an error, but it is silent.

---

Thanks for checking, I can look into it.

---

It seems using type `string` with a date-like formatting wasn't working. I switched to using a regex pattern constraint, which will catch non-dates but still allows scenarios like fewer than 3 dates and null values. The regex syntax is a little weird—for example, I had to put a real whitespace instead of `\s?`